addons/website/views/snippets/s_cover.xml

```diff
@@ -2,7 +2,7 @@
 <odoo>
 
 <template id="s_cover" name="Cover">
-    <section class="s_cover parallax s_parallax_is_fixed bg-black-50 pt96 pb96" data-scroll-background-ratio="1">
+    <section class="s_cover parallax s_parallax_is_fixed o_colored_level o_cc o_cc3 pt96 pb96" data-scroll-background-ratio="1">
```

Comment:
For this snippet and the s_parallax one, using bg-black-50 was actually a well-thought choice: there is an image on top of that background color and there is a black-50 background filter on top of it. Using black-50 for the background class actually makes it so the title are automatically white to go well with that bg-black-50 filter on top of the image.

With this change, you actually "create a bug":
- Go to the theme tab in edit mode
- Change the color of the background of o_cc3 to a light color (everything is adapted automatically: o_cc3's text should now be black)
- Add a cover snippet -> you text is black over a black filter (sure the user can change it but the default is not nice).

I am not sure what the proper fix should be, maybe those 2 snippets should stay with the UX issue your commit tries to fix, let's ask @pierre-pde